piff/optical_model.py

```diff
@@ -17,41 +17,69 @@
 .. module:: optical_model
 """
 
+from __future__ import print_function
+
 import galsim
 import coord
 import fitsio
 import numpy as np
+from functools import lru_cache
 
 from .model import Model
 from .star import Star, StarFit, StarData
 
-# The only one here by default is 'des', but this allows people to easily add another template
 optical_templates = {
-    'des': { 'obscuration': 0.301 / 0.7174,
+    'des_simple': {'obscuration': 0.301 / 0.7174, # from Zemax DECam model
              'nstruts': 4,
-             'diam': 4.274419,  # meters
+             'diam': 4.010,  # meters
              'lam': 700, # nm
-             # aaron plays between 19 mm thick and 50 mm thick
-             'strut_thick': 0.050 * (1462.526 / 4010.) / 2.0, # conversion factor is nebulous?!
+             'strut_thick': 3.5 * 0.019 * (1462.526 / 4010.) / 2.0,  #updated to match big DES donuts, ignores one larger spider leg
```

Comment:
These are taken from my *donut* code, tuned to match real images of big Donuts.  I kept it as this combination to remember where it came from, but it could certainly just be kept as the final value with the comment that it was tuned to match data.

Reply:
Ah crap - these have probably always been wrong for Galsim - I will fix...